app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -58,6 +59,11 @@ public function register(RegisterRequest $request): JsonResponse
             'store_databases' => $this->settings->get($prefix . 'database', 0),
         ];
 
+        if (!$this->settings->get($prefix . 'verification')) {
```

Comment:
Might as well just have

```
$data['verified'] = true;

if ($this->settings->get($prefix . 'verification') != 'true') $data['verified'] = false;
```

Also, when retrieving data from SettingsRepositoryInterface, check like `!= 'true'` as value is stored as str. Trust me, things go wrong if you don't.